Package.swift

```diff
@@ -161,12 +161,12 @@ let package = Package(
       "2.30909.0" ..< "2.30910.0"
     ),
     .package(
-      url: "https://github.com/firebase/abseil-cpp-SwiftPM.git",
-      "0.20220203.1" ..< "0.20220204.0"
+      url: "https://github.com/google/abseil-cpp-binary.git",
+      "1.2021110200.0" ..< "1.2021110300.0"
```

Comment:
So we were previously using different versions of abseil between CocoaPods and SPM?  

Perhaps for the next release, we should update to 20220203?

Reply:
Yep, I noticed that. This version is coming from the zip README, which is coming from here: https://github.com/firebase/firebase-ios-sdk/blob/98e218b68643e8b02cbe31f5de53fe1e04c23d74/FirebaseFirestore.podspec#L93

Reply:
Discussed offline. I'll look at aligning the versions between CP and SPM. cc: @wu-hui